test/contracts/apps/recovery_to_vault.js

```diff
@@ -17,22 +20,53 @@ const TokenReturnFalseMock = artifacts.require('TokenReturnFalseMock')
 const TokenReturnMissingMock = artifacts.require('TokenReturnMissingMock')
 const VaultMock = artifacts.require('VaultMock')
 const KernelDepositableMock = artifacts.require('KernelDepositableMock')
+const EthSender = artifacts.require('EthSender')
 
 const APP_ID = hash('stub.aragonpm.test')
 const EMPTY_BYTES = '0x'
-const SEND_ETH_GAS = 31000 // 21k base tx cost + 10k limit on depositable proxies
+const TX_BASE_GAS = 21000
+const SEND_ETH_GAS = TX_BASE_GAS + 10000 // 10k limit on depositable proxies
+const SOLIDITY_TRANSFER_GAS = 2300
+const ISTANBUL_SLOAD_GAS_INCREASE = 600
 
-contract('Recovery to vault', ([permissionsRoot]) => {
-  let aclBase, appBase, appConditionalRecoveryBase
+contract('Recovery to vault', ([ permissionsRoot ]) => {
+  let aclBase, appBase, appConditionalRecoveryBase, ethSender
   let APP_ADDR_NAMESPACE, ETH
 
   // Helpers
+  const sendEth = async ({ target, value, gas = SOLIDITY_TRANSFER_GAS, shouldOOG }) => {
```

Comment:
Perhaps we can break down this test helper a bit... right now there's a bit too much logic here.